plugins/lua/eventful.lua

```diff
@@ -164,6 +164,7 @@ eventType=invertTable{
     "UNIT_ATTACK",
     "UNLOAD",
     "INTERACTION",
+    "PRESAVE",
     "EVENT_MAX"
```

Comment:
The compiler was giving me a hard time because some other header conflicted with it. It could probably be reverted. I'll try it when I get a chance.